src/webapp/topology.py

```diff
@@ -217,6 +246,15 @@ def get_tree(self, authorized=False, filters: Filters = None) -> Optional[Ordere
 
         return new_res
 
+    @property
+    def is_ccstar(self):
+        """Check if this site is tagged as a CC* Site"""
+        try:
+            return "CC*" in self.data["Tags"]
+        except:
+            return False
```

Comment:
```suggestion
        return "CC*" in self.data.get("Tags", [])
```
Let's avoid an overly broad except.

Reply:
@matyasselmeci Oh awesome, this a lot better!